src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -460,6 +468,7 @@ def load_arguments(self, _):
                    help='Provide a string value of a Storage Account in the provided Resource Group. Or Resource ID of a Storage Account in a different Resource Group')
         c.argument('consumption_plan_location', options_list=['--consumption-plan-location', '-c'],
                    help="Geographic location where Function App will be hosted. Use `az functionapp list-consumption-locations` to view available locations.")
+        c.argument('version', options_list=['--version', '-v'], help='The functions runtime version.', arg_type=get_enum_type(HOST_VERSIONS_FUNCTIONAPP), configured_default='2')
```

Comment:
Maybe more description about version 2 and 3. I don't know which one I should choose.